pkg/container-collection/containers.go

```diff
@@ -67,6 +70,13 @@ type Container struct {
 	ownerReference *metav1.OwnerReference
 }
 
+type ContainerEvent struct {
+	Timestamp string `json:"timestamp,omitempty" column:"timestamp,maxWidth:30" columnTags:"runtime"`
+	Event     string `json:"event,omitempty" column:"event,maxWidth:10" columnTags:"runtime"`
+
+	Container
+}
```

Comment:
How about using https://github.com/inspektor-gadget/inspektor-gadget/blob/3429862cec69552f890c975c2d3817a13b737ab0/pkg/container-collection/pubsub.go#L30-L33

Reply:
I had a look into this and we need add support for handling pointers to struct in `columns` library. Is this a use case we want to support? 

cc: @flyth